content/all_datasets.md

```diff
@@ -6,56 +6,57 @@ full-width: true
+| [AIT Log Dataset](../datasets/ait_log_dataset)                                                     |       Both        | Huge variety of labeled logs collected from multiple simulation runs of an enterprise network under attack. With user emulation. but only Linux machines             |      2022 | Enterprise IT | Linux                 |    🟩     | pcaps, Suricata alerts, misc. logs (Apache, auth, dns, vpn, audit, suricata, syslog) |      130 GB |        206 GB |
+| [CLUE-LDS](../datasets/clue_lds)                                                                   |       Host        | Database of real user behavior without known attacks, for evaluation of methods detecting shifts in user behavior                                                    |      2022 | Subsystem     | Undisclosed           |    🟥     | Custom event logs                                                                    |      640 MB |       14,9 GB |
+| [EVTX to MITRE ATT&CK](../datasets/evtx_to_mitre_attck)                                            |       Host        | Small dataset providing various events corresponding to certain MITRE tactics/techniques                                                                             |      2022 | Single OS     | Windows               |    🟩     | Windows events                                                                       |       <1 GB |         <1 GB |
+| [OTFR Security Datasets - Atomic](../datasets/otfr_atomic)                                         |       Both        | Various small datasets, each corresponding to a specific MITRE tactic/technique. Lacks user simulation / underlying scenario and does not provide explicit labels    | 2019-2022 | Single OS     | Windows, Linux, Cloud |    🟨     | pcaps, Windows events, auditd logs, AWS CloudTrail logs                              |      125 MB |             - |
+| [PWNJUTSU](../datasets/pwnjutsu)                                                                   |       Both        | Rich collection of complex attacks executed by various red team participants each acting in a small network, but not labeled                                         |      2022 | Miscellaneous | Windows, Linux        |    🟥     | pcaps, Windows events, Sysmon, auditd, various logs (Apache, auth, dns, ssh, etc.)   |       82 GB |             - |
+| [UWF-ZeekData22](../datasets/uwf_zeekdata22)                                                       |      Network      | Traffic collected from a universities wargaming course. Covers all MITRE tactics, though the overwhelming majority is simple recon and attacks are poorly documented |      2022 | Enterprise IT | Windows, Linux        |    🟩     | pcaps, Zeek logs                                                                     |           - |        209 GB |
```

Comment:
We could and markdown allows for it, but I'd prefer not to do that because it makes editing somewhat annoying (for me). It should happen rarely anyways (this was the first time since we've started adding new datasets). I'll change it if you insist 😄 the line marked here is the one that was actually added

Reply:
thx, sure we can leave it if it's easier for you.